scripts/consent_form_json.py

```diff
@@ -1,24 +1,90 @@
+""" Download all csv files with consent form content from google drive and combine into a single json file.
+    Requires a client_secret.json file with developer application credentials from google drive.
+    Modifies: https://developers.google.com/drive/v3/web/quickstart/python """
+
+import httplib2
+import os
 import csv
 import json
-import collections
-import os.path
 
-# Content for each site's consent form
-files = os.listdir('/Users/Saman/Devel/isp/scripts/consent_forms/')
+from apiclient import discovery
+import oauth2client
```

Comment:
Can these be unified into a single import? Either
- `import oauth2client` only; or
- `from oauth2client import file, client, tools`